src/components/DatePickerInline/Month.tsx

```diff
@@ -70,68 +70,50 @@ function Month(props: MonthSingleProps | MonthRangeProps | MonthMultiProps) {
         [year, month, index, isDisabled, mode, isWithinValidRange, startDate, endDate, dates, date],
     );
 
-    const {
-        monthStyle,
-        headerStyle,
-        yearButtonStyle,
-        yearInnerStyle,
-        monthLabelStyle,
-        iconContainerStyle,
-    } = useMemo(() => {
-        const {
-            monthLabel: _monthLabel,
-            yearButton,
-            yearButtonInner,
-            month: _monthStyle,
-            monthHeader,
-        } = monthStyles;
-        const { typescale, ...monthLabel } = _monthLabel;
-
-        return {
-            monthStyle: [_monthStyle, { height: getMonthHeight(scrollMode, index) }],
-            headerStyle: [
+    const { headerStyle, yearButtonStyle, yearInnerStyle, monthLabelStyle, weekContainerStyle } =
+        useMemo(() => {
+            const {
+                monthLabel: _monthLabel,
+                yearButton,
+                yearButtonInner,
+                month: _monthStyle,
                 monthHeader,
-                isHorizontal
-                    ? {
-                          marginTop: monthHeaderSingleMarginTop,
-                          marginBottom: monthHeaderSingleMarginBottom,
-                      }
-                    : null,
-            ],
-            yearButtonStyle: yearButton,
-            yearInnerStyle: yearButtonInner,
-            monthLabelStyle: [monthLabel, typescale],
-            iconContainerStyle: { opacity: isHorizontal ? 1 : 0 },
-        };
-    }, [index, isHorizontal, monthStyles, scrollMode]);
-
-    const onPressDropdown = useCallback(
-        () => (isHorizontal && onPressYear ? onPressYear(year) : undefined),
-        [isHorizontal, onPressYear, year],
-    );
+            } = monthStyles;
+            const { typescale, ...monthLabel } = _monthLabel;
+
+            return {
+                monthStyle: [_monthStyle, { height: getMonthHeight(scrollMode, index) }],
+                headerStyle: [
+                    monthHeader,
+                    isHorizontal
+                        ? {
+                              alignItems: isDocked && 'flex-start',
```

Comment:
remove conditional logic, make parent component passdown the data as required.

Reply:
removed @shashank1010